strings/tips/xeno.txt

```diff
@@ -38,3 +38,28 @@ As a spitter, boiler, praetorian, or the Queen take advantage of unguarded barri
 As a boiler or the Queen, you have the most powerful corrosive acid for melting things.
 As the Shrike, take advantage of your psychic abilities. Use Psychic Fling and Unrelenting Force to displace a marine or even push certain and hazardous objects to your enemies and vice versa.
 As the Queen, your screech does not only affect the mobs on view, but affects through walls or gases. However, the effect of the screech will be reduced.
+As a drone, hivelord, hivemind, shrike, and queen, large maze is almost always a good investment.
```

Comment:
Determiners come before the noun. "**_A_** large maze is almost always a good investment" would suffice.
- - -
I see room for another improvement. "As a drone, hivelord, hivemind, shrike, **_or_** queen," is easier to read.